daemon/src/main.rs

```diff
@@ -192,3 +200,12 @@ fn ask(question: &str) -> Result<bool> {
         bail!("Failed to read answer");
     }
 }
+
+fn has_git_remote(path: &Path) -> bool {
```

Comment:
It's not necessarily config, but I could also see this in the `config.rs` module:
- it's about inferring the "configured" behaviour of the application
- we could, later, think about introducing a variable to influence/overwrite the inferred use-case
- if we do this, `AppConfig` could also get a `persist` field